airbyte-webapp/src/components/connectorBuilder/useManifestToBuilderForm.ts

```diff
@@ -60,7 +61,7 @@ export const convertToBuilderFormValues = async (
   }
   const resolvedManifest = resolveResult.manifest as ConnectorManifest;
 
-  const builderFormValues = DEFAULT_BUILDER_FORM_VALUES;
+  const builderFormValues = cloneDeep(DEFAULT_BUILDER_FORM_VALUES);
```

Comment:
Fixes a bug where the lines below this one were actually updating the `DEFAULT_BUILDER_FORM_VALUES` constant's fields, causing issues like `Reset All` not behaving properly